pkgs/build-support/cc-wrapper/fortran-hook.sh

```diff
@@ -0,0 +1,11 @@
+getTargetRole
+getTargetRoleWrapper
+
+export FC${role_post}=@named_fc@
+
+# If unset, assume the default hardening flags.
+# These are different for fortran.
+: ${NIX_HARDENING_ENABLE="stackprotector pic strictoverflow relro bindnow"}
+export NIX_HARDENING_ENABLE
```

Comment:
Is this fortran-specific?

Reply:
yeah - fortify & format don't work well in fortran. it's non-fatal, but we shouldn't need to set it.

Reply:
i guess this is kind of an issue since it does effect cc/c++ as well